src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -350,6 +368,8 @@ private BoundLambda ReallyBind(NamedTypeSymbol delegateType)
             }
 
             var parameters = DelegateParameters(delegateType);
+            CheckDelegateDynamicParametercs(parameters, diagnostics);
```

Comment:
It feels like the lack of attributes is going to affect result of overload resolution. Should it? Were attributes required by the native compiler? If not, we probably should make them optional for lambda parameters.